src/button/interfaces.ts

```diff
@@ -100,6 +100,17 @@ export interface BaseButtonProps {
    * @i18n
    */
   i18nStrings?: ButtonProps.I18nStrings;
+
+  /**
+   * Attributes to add to the native `button` element.
+   *
+   * It is not supported to use this attribute to apply custom styling.
```

Comment:
What does "not supported" actually mean? Technically, style property can still be used.

Reply:
it means we don't provide any promises that it will not break in the future if you use it for styling.

Reply:
Should we say something like: "Do not use this property to provide custom inline styles to the component"?